2/contrib/s2i/run

```diff
@@ -78,10 +90,13 @@ function install_plugins() {
     /usr/local/bin/install-plugins.sh "${plugins_file}"
   fi
   
-  if [ "$(ls -A /opt/openshift/plugins 2>/dev/null)" ]; then
-    mkdir -p ${JENKINS_HOME}/plugins
-    echo "Copying $(ls /opt/openshift/plugins | wc -l) Jenkins plugins to ${JENKINS_HOME} ..."
-    cp -r /opt/openshift/plugins/* ${JENKINS_HOME}/plugins/
+  if [ "$(ls /opt/openshift/plugins/*.jpi 2>/dev/null)" ]; then
+    echo "Copying $(ls /opt/openshift/plugins/*.jpi | wc -l) Jenkins plugins to ${JENKINS_HOME} ..."
+    # since install_plugins populates this, the files already end in jpi, don't need to swap suffixes
```

Comment:
well, it's not just install_plugins that populates it:
https://github.com/openshift/jenkins/blob/master/2/contrib/s2i/assemble#L3
https://github.com/openshift/jenkins/blob/master/2/contrib/s2i/assemble#L25

so if someone does an s2i build using this as the builder image, and they have a .hpi file in their source plugins dir, it's going to get lost here.

Reply:
seems safer to just continue to check if plugins is empty or not.

Reply:
what was the motivation to narrow the check?

Reply:
simply trying to prevent unintended side effects from unintended copying of files

given the assemble discovery, I'm retracting ... other than incorporating the `-A` comment from @jim-minter I'll just copy everything

if something pops up from copying something we shouldn't we'll handle then

Reply:
adjusting the echo as well